_includes/manuals/nightly/2_quickstart_guide.md

```diff
@@ -4,11 +4,14 @@ The Foreman installer is a collection of Puppet modules that installs everything
 Components include the Foreman web UI, Smart Proxy, Passenger, a Puppet master (either Puppet Server or under Passenger), and optionally TFTP, DNS and DHCP servers.  It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
 
 #### Supported platforms
-* CentOS, Scientific Linux or Oracle Linux 7, x86_64
+* CentOS 7 x86_64
 * Debian 10 (Buster), i386/amd64/aarch64
 * Red Hat Enterprise Linux 7, x86_64
 * Ubuntu 18.04 (Bionic), i386/amd64/aarch64
 
+#### Untested platforms(packages may not work on these)
+* Scientific Linux and Oracle Linux
```

Comment:
How about:
```suggestion
#### Untested platforms
These platforms are not tested by automatic installations. They are generally close to supported platforms so the packages may work, but additional work may be needed.
* Scientific Linux and Oracle Linux 7, x86_64
```

Reply:
If we say 'additional work may be needed' but wont specify what exactly will be required ? wont it lead to more questions ? like what and where ?

Reply:
I also thought about that but couldn't find a good answer. There have been discussions on discourse: https://community.theforeman.org/search?q=oracle%20linux. Perhaps we should link to the discourse support forum for community support?

Reply:
what's the decision here?

Reply:
How about this:
These platforms are not tested by automatic installations. They are generally close to supported platforms so the packages may work, but additional work may be needed. For any queries for these platforms raise a question in [discourse support section](https://community.theforeman.org/c/support/10)
* Scientific Linux and Oracle Linux 7, x86_64

Reply:
That sounds good to me.